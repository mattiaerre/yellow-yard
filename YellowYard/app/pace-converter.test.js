import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import App from './pace-converter';

describe('<App />', () => {
  test('expect to match snapshot', () => {
    const tree = renderer.create(
      <App />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
