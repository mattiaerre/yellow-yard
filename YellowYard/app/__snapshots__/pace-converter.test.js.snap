// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> expect to match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5F5F5",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Menlo-Bold",
        },
        Object {
          "fontSize": 30,
          "margin": 10,
        },
      ]
    }
  >
    Pace converter
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Menlo",
        },
        Object {
          "fontSize": 15,
          "margin": 10,
        },
      ]
    }
  >
    Universal running pace converter.
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Menlo",
        },
        Object {
          "fontSize": 20,
          "margin": 10,
        },
      ]
    }
  >
    min/km
  </Text>
  <TextInput
    onChangeText={[Function]}
    style={
      Array [
        Object {
          "fontFamily": "Menlo",
        },
        Object {
          "fontSize": 20,
          "margin": 10,
        },
        Object {
          "alignSelf": "center",
          "borderColor": "gray",
          "borderWidth": 1,
          "height": 40,
          "textAlign": "center",
          "width": 100,
        },
      ]
    }
    value="4:45"
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Menlo",
        },
        Object {
          "fontSize": 20,
          "margin": 10,
        },
      ]
    }
  >
    km/h 
    12.632
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Menlo",
        },
        Object {
          "fontSize": 20,
          "margin": 10,
        },
      ]
    }
  >
    min/mi 
    7:38.64
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Menlo",
        },
        Object {
          "fontSize": 20,
          "margin": 10,
        },
      ]
    }
  >
    mi/h 
    7.849
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Menlo",
        },
        Object {
          "fontSize": 10,
          "margin": 10,
        },
      ]
    }
  >
    v1.0.0-alpha.1
  </Text>
</View>
`;
